src/graphnet/models/cnn/theos_muonE_upgoing.py

```diff
@@ -0,0 +1,411 @@
+"""CNN used for muon energy reconstruction in IceCube.
+
+Copy of `upgoing_muon_energy` model from
+https://github.com/IceCubeOpenSource/i3deepice/tree/master
+
+Class and variable names are kept for
+compatibility with the original code.
+"""
+
+from typing import Tuple, Union
+
+import torch
+from torch import nn
+from pytorch_lightning import LightningModule
+from torch_geometric.data import Data
+from .cnn import CNN
+
+
+class Conv3dBN(LightningModule):
+    """The Conv3dBN module from Theos CNN model."""
```

Comment:
Theos -> Theo Glauch

Reply:
Consider adding a bit more detail to inform the reader of what this module is. E.g. 

`"""Implementation of the Conv3dBN image convolution module from Theo Glauch.""" `